spec-tests/src/main/java/org/candlepin/spec/bootstrap/data/util/ExportUtil.java

```diff
@@ -0,0 +1,133 @@
+     * @throws IOException if unable to create a temporary export zip file.
+     */
+    public static ZipFile getExportArchive(File manifest) throws IOException {
+        if (manifest == null) {
+            throw new IllegalArgumentException("Manifest is null.");
+        }
+
+        try (ZipFile zipFile = new ZipFile(manifest)) {
+            ZipEntry entry = zipFile.getEntry(EXPORT_NAME);
+            File tmp = File.createTempFile("export", ".zip");
+            tmp.deleteOnExit();
+            try (InputStream istream = zipFile.getInputStream(entry);
+                FileOutputStream ostream = new FileOutputStream(tmp)) {
+                istream.transferTo(ostream);
+            }
+
+            return new ZipFile(tmp);
+        }
+    }
+
+    /**
+     * Retrieves all {@link ZipEntry}s in a provided path inside of the provided {@link ZipFile}.
+     *
+     * @param path location of the directory to retrieve zip entries from.
+     * @param zipFile the zip file system to retrieve the zip entries from.
+     * @return the zip entries in the provided zip file and target directory.
+     */
+    public static List<ZipEntry> getFilesInDirectory(String path, ZipFile zipFile) {
+        List<ZipEntry> foundFiles = new ArrayList<>();
+        Enumeration<? extends ZipEntry> zipEntries = zipFile.entries();
+        while (zipEntries.hasMoreElements()) {
+            ZipEntry entry = zipEntries.nextElement();
+            if (entry.getName().contains(path)) {
+                foundFiles.add(entry);
+            }
+        }
+
+        return foundFiles;
+    }
+
+    /**
+     * Get a single {@link ZipEntry} that matches the provided file name in a given {@link ZipFile}.
+     *
+     * @param fileName the name of the file to retrieve.
+     * @param zipFile the zip file to retrieve the {@link ZipEntry} from.
+     * @return a zip entry that matches the provided file name.
+     */
+    public static ZipEntry getFile(String fileName, ZipFile zipFile) {
+        Enumeration<? extends ZipEntry> entries = zipFile.entries();
+        while (entries.hasMoreElements()) {
+            ZipEntry entry = entries.nextElement();
+            // The entry name contains the file path, so only compare with the end.
+            if (entry.getName().endsWith(fileName) && !entry.isDirectory()) {
+                return entry;
+            }
+        }
+
+        return null;
+    }
+
+    /**
+     * Copies the provided {@link ZipEntry}s to temporary {@link File}s that can be used to read content from.
+     *
+     * @param zipFile the zip file to read zip entries from.
+     * @param entries the individual files that should be copied to temporary files.
+     * @return a map of the zip entry file name to the corresponding temporary file.
+     * @throws IOException if unable to read from the zip entries or write to the new files.
+     */
+    public static Map<String, File> copyToTempFiles(ZipFile zipFile, Collection<ZipEntry> entries)
```

Comment:
I see what you are saying, using the FileInputStream instead of creating all these temp files makes life so much better. I removed this method since it's not needed anymore.